common/models/key-value-model.js

```diff
@@ -1,13 +1,13 @@
-var g = require('../../lib/globalize');
-
 /**
  * Data model for key-value databases.
  *
  * @class KeyValueModel
  * @inherits {Model}
  */
+ 'use strict';
+ var g = require('../../lib/globalize');
 
-module.exports = function(KeyValueModel) {
+ module.exports = function(KeyValueModel) {
```

Comment:
The indentation changes in this file are suspicious. Why do we need to indent lines by one (!) more character? AFAIK, we are not doing that in any other file. PTAL.